pkg/devfile/adapters/common/utils.go

```diff
@@ -128,6 +128,8 @@ func getCommandsByGroup(data data.DevfileData, groupType common.DevfileCommandGr
 	for _, command := range data.GetCommands() {
 		if command.Exec != nil && command.Exec.Group != nil && command.Exec.Group.Kind == groupType {
 			commands = append(commands, command)
+		} else if command.Composite != nil && command.Composite.Group != nil && command.Composite.Group.Kind == groupType {
```

Comment:
if and else-if condition can be combined here too?

Reply:
Done